suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -280,7 +280,10 @@ export const selectAccountByKey = createMemoizedSelector(
     },
 );
 
-export const selectHasAccountTransactions = createMemoizedSelector(
+// CAUTION!: This selector is not working for XRP accounts!
+// Ripple backend does not provide the total transaction count info.
+// The property`account.history.total` is always equal to -1 for XRP accounts.
+export const selectHasNonRippleAccountTransactions = createMemoizedSelector(
```

Comment:
The name of the selector is misleading... I'd prefer something like `selectHasAccountAnyTransactionHistory` or similar.